Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+
+* **Scan: QRs.** Photograph QRs of SSKR shares until you have a threshold.
+* **Scan: Paste Crypto-SSKR.** Paste `ur:crypto-sskr` of SSKR shares until you have a threshold.
+* **Add: Shares.** Simultaneouly paste sufficient shares to meet a threshold into the box.
+
+The `ur:crypto-sskr` specification, which can be used in the `Add` section, in the `Scan: Paste`, or encoded as a QR, look like this:
+
+* ur:crypto-sskr/gobnbdaeadaevdbkclhseeehtldedikgpysoenreceeeeorofrwn
+* ur:crypto-sskr/gobnbdaeadadrkhesefhjzdycypduokkjejponsayabguymwwnwz
+
+The SSKR words, which can only be used in the `Add` section, look like this:
+
+* tuna acid epic gyro gray tent able acid able frog cusp heat poem owls undo holy rich monk zaps cost brag loud fair dice idle skew iris beta tied
+* tuna acid epic gyro gray tent able acid acid diet fact gala numb leaf fish toys kite cyan inky help keep heat inky song trip bulb flap yoga jazz
+
+The **Scan** functionality is currently the more advanced of the two options, and so is the suggested methodology. It will allow you to photograph or paste individual shares, and will alert you to how many more are needed to meet the threshold and reconstruct the seed.
+
+### Creating a Seed
+
+**Gordian Seed Tool** can also be used to create new seeds. This is doing using the **add** (+) button on the main menu. There are four ways to do so:
+
+* **Quick Create.** Depend on your mobile device's randomization to create a seed.
+* **Coin Flips.** Flip coins and enter results.
+* **Die Rolls.** Roll six-sided dice and enter results.
+* **Card Draws.** Draw cards and enter results. (Be sure to replace cards and reshuffle as you draw, for the entropy calculation to be correct.)
+
+The easiest of these methods is certainly the "Quick Create", but in that case you are depending on the randomization of your mobile device, and if there is ever an exploit revealed, you'll be forced to sweep all of your funds. Using coin flips, die rolls, or playing cards is perhaps more likely to create good entropy, and is less likely to have an exploit revealed, but you _must_ properly flip every coin, roll every die, or draw every card, no matter how tedious it is, and you must wait until you have at least 128 bits of entropy, which is considered "Very Strong". If you are not willing to do this, you should just "Quick Create" instead. Once you have a "Very Strong" amount of entropy, you should click the "Done" button, and then you'll be given the opportunity to "Save" your new seed.
+
+The coin flipping, die rolling, and card drawing methods all have three buttons at the bottom, which allow you to: erase one entry; use the randomizer to add one entry; or use the randomizer to add all of the entries.
+
+The usage of coin, die, and card entropy in **Gordian Seed Tool** matches that of [**LetheKit**](https://github.com/BlockchainCommons/lethekit), so you can check the results for one on the other if you want to be sure of a new seed that you've created.
```

Comment:
Better known then checking against seedtool-cli or Lethekit, is Ian Coleman's https://iancoleman.io/bip39/ website, if you want to compare results to demonstrate "nothing up our sleeves" generation of keys from randomness.

Reply:
I have not personally verified that the transformations from entropy to seed that Seed Tool uses are identical to those used by Lethekit. The "coin flips" and "die rolls" method is identical to Ian Coleman's BIP-39 site, but the "card draws" technique is definitely *not* identical to Ian's site.

Reply:
I've instead linked to Ian's site and noted the card draws aren't the same.